ssg/build_yaml.py

```diff
@@ -539,6 +547,30 @@ def from_yaml(yaml_file, env_yaml=None):
         rule.validate_references(yaml_file)
         return rule
 
+    def _set_attributes_from_dict(self, yaml_contents):
+        for key, default_getter in self.YAML_KEYS_DEFAULTS.items():
+            if key not in yaml_contents:
+                value = default_getter()
+                if isinstance(value, Exception):
+                    raise value
+            else:
+                value = yaml_contents.pop(key)
+
+            setattr(self, key, value)
+
+    def to_contents_dict(self):
+        """
+        Returns a dictionary that is the same schema as the dict obtained when loading rule YAML.
+        """
+
+        yaml_contents = dict()
+        for key in Rule.YAML_KEYS_DEFAULTS:
+            yaml_contents[key] = getattr(self, key)
+
+        yaml_contents["documentation_complete"] = True
```

Comment:
Will this value be used later?
I think at this point, all rules are supposed to be picked up for build.

Reply:
I was not sure about that. I guess that it can be safely removed.